src/main/java/net/minestom/server/network/socket/Server.java

```diff
@@ -112,8 +116,21 @@ public void stop() {
         } catch (IOException e) {
             MinecraftServer.getExceptionManager().handleException(e);
         }
-        this.selector.wakeup();
```

Comment:
I'm unconvinced about removing these, it seems valid to process anything pending before shutting down

Reply:
Close calls this internal also